frontend/pages/compensators/[documentId].js

```diff
@@ -0,0 +1,498 @@
+                  <h2 className="text-gray-800 font-bold mb-4 border-b border-blue-100 pb-2">
+                    WATER COMPENSATION PLEDGE:
+                  </h2>
+                  <div className="mb-4">
+                    <div className="flex justify-between mb-2">
+                      <div>
+                        <div className="text-gray-500 text-sm font-bold">
+                          ACTUAL
+                        </div>
+                        <div className="text-[#0DA2D7] font-bold">
+                          {compensator.actualPledgePercentage}%
+                        </div>
+                      </div>
+                      <div className="text-right">
+                        <div className="text-gray-500 text-sm font-bold">
+                          TARGET
+                        </div>
+                        <div className="text-gray-700 font-bold">100%</div>
+                      </div>
+                    </div>
+                    <div className="h-6 bg-gray-200 rounded-md overflow-hidden">
+                      <div
+                        className="h-full bg-[#0DA2D7] rounded-md"
+                        style={{
+                          width: `${compensator.actualPledgePercentage}%`,
+                        }}
+                      ></div>
+                    </div>
+                  </div>
+                </div>
+              </div>
+
+              <div className="md:w-2/5">
+                <h2 className="text-gray-800 font-bold mb-4">SDGS:</h2>
+                <div className="flex justify-center">
+                  <SDGWheel
+                    sdgData={compensator.sdgs?.map((sdg) => ({
+                      id: sdg.id,
+                      title: sdg.name,
+                      color: sdg.colour,
+                      icon: sdg.icon
+                        ? () => (
+                            <Image
+                              src={`${env('NEXT_PUBLIC_BACKEND_URL')}${
+                                sdg.icon.url
+                              }`}
+                              alt={sdg.name}
+                              className="w-full h-full object-cover"
+                              width={100}
+                              height={100}
+                              unoptimized
+                            />
+                          )
+                        : sdgData.find(
+                            (defaultSDG) => defaultSDG.title === sdg.name
+                          )?.icon || TreePine,
+                    }))}
+                  />
+                </div>
+              </div>
+            </div>
+            {(compensator?.compensationProgressDescription ||
+              compensator?.compensationProgressImage?.url) && (
+              <div>
+                <div>
+                  <h2 className="text-gray-800 font-bold mb-4 border-t border-blue-100 mt-6 pt-6">
+                    SOFTA WATER FOOTPRINT COMPENSATION & REDUCTION 5 YEAR
```

Comment:
Remove this hard coded text. Let's add a `compensationProgressTitle` to the data model.